Packs/CommonScripts/Scripts/ExtractIndicatorsFromWordFile/ExtractIndicatorsFromWordFile.py

```diff
@@ -31,7 +31,7 @@ def get_file_details(self):
         else:  # Got file path:
             self.file_path = file_res.get('Contents').get('path')
             self.file_name = file_res.get('Contents').get('name')
-            file = demisto.dt(demisto.context(), "File(val.EntryID === '{}')".format(demisto.args().get('entryID')))
+            file = demisto.dt(self.get_context(), "File(val.EntryID === '{}')".format(demisto.args().get('entryID')))
```

Comment:
please switch line 26 to `demisto.getFilePath` instead of `demisto.executeCommand`